### PR TITLE
Ctypes: allow analysis on incomplete graphs

### PR DESCRIPTION
**Pull request by serpilliere**

### CODE REVIEW

File: example/ida/ctype_propagation.py [hunk 1]
```diff
@@ -255,7 +255,8 @@ def analyse_function():
         done.add((lbl, state))
         symbexec_engine = TypePropagationEngine(ir_arch, types_mngr, state)
 
-        assert lbl in ir_arch.blocks
+        if lbl not in ir_arch.blocks:
```
Comment (commial) [inline example/ida/ctype_propagation.py]: If you're going to continue (and skip the rest of the loop), there is no need to instance a new symbexec engine (ie, the `if` statement should be upper)

File: example/ida/ctype_propagation.py [hunk 2]
```diff
@@ -267,6 +268,8 @@ def analyse_function():
 
     for lbl, state in states.iteritems():
         symbexec_engine = CTypeEngineFixer(ir_arch, types_mngr, state, cst_propag_link)
+        if lbl not in ir_arch.blocks:
```
Comment (commial) [inline example/ida/ctype_propagation.py]: Idem than L.258

File: miasm2/analysis/cst_propag.py [hunk 3]
```diff
@@ -141,7 +141,8 @@ def compute_cst_propagation_states(ir_arch, init_addr, init_infos):
         done.add((lbl, state))
         symbexec_engine = SymbExecState(ir_arch, state)
 
-        assert lbl in ir_arch.blocks
+        if lbl not in ir_arch.blocks:
```
Comment (commial) [inline miasm2/analysis/cst_propag.py]: Idem than for `example/ida/ctype_propagation.py`, for L.144 and L.178

**Comment by serpilliere** [comment]
Done!

**Pull request merged**